frontend/src/modules/landing/new-home.jsx

```diff
@@ -183,6 +184,11 @@ const Landing = ({
   };
 
   useEffect(() => {
+    if (!data) {
```

Comment:
if we move this call inside this file and don't update the store then we couldn't reuse this data from different views, do we?

Reply:
I check for where's the `data` used, 
Then I found the `data` only used on the landing page, so I decided to move that to the landing page instead of fetching in on root. Do I need to revert this change?

Reply:
if this data is only used in landing page, then IMO this is the place where it should be the logic in charged of fetching it